build/emlib.rs

```diff
@@ -110,9 +110,7 @@ fn prod_config(config: &mut Config) -> &mut Config {
         .file("src/leuart/get_leuart.c")
 
         .include("efm32-common/kits/common/drivers")
```

Comment:
Yeah, I was a bit unsure about this one as it is used by [`src/dma/dma.c`](https://github.com/RustyGecko/emlib/blob/master/src/dma/dma.c). Should we remove it and require the user to opt into it by another library ?